x-pack/plugins/infra/public/pages/metrics/index.tsx

```diff
@@ -124,7 +129,18 @@ export const MetricDetail = withTheme(
                             >
                               {({ metrics, error, loading, refetch }) => {
                                 if (error) {
-                                  return <ErrorPageBody message={error} />;
+                                  if (error.graphQLErrors) {
```

Comment:
According to the type this will always be an array. Won't this be `true` even if the array is empty? Or is the type incorrect?